app.js

```diff
@@ -240,17 +258,50 @@ function contactValidation (e) {
   }
 }
 
-contactValidation()
-=======
-</div>`;
-  modalDisplay.innerHTML = display;
-};
+</div>`
+      modalDisplay.innerHTML = display
+    }
 
-openMenu.addEventListener('click', show);
+openMenu.addEventListener('click', show)
 
-closeMenu.addEventListener('click', close);
-closeModalButton.addEventListener('click', closeModal);
+closeMenu.addEventListener('click', close)
+closeModalButton.addEventListener('click', closeModal)
 openModalButton.forEach((item, index) => {
-  item.addEventListener('click', openModal(arrayData[index]));
-});
->>>>>>> 8f844131951652dc5de3654db9b15ee23df4dae2
+
+  item.addEventListener('click', openModal(arrayData[index]))
+})
+
+function contactValidation (e) {
+  const email = document.getElementById('email').value
+
+  if (email === '') {
+    e.preventDefault()
+
+    small.innerHTML = 'e-mail field cannot be emty'
+  } else if (email !== email.toLowerCase()) {
+    e.preventDefault()
+
+    small.innerHTML =
+      'email must be lowercase'
+  } else {
+    small.innerHTML = 'FORM submitted!'
+    small.style.color = '#00ff00'
+  }
+}
+
+formSubmit.addEventListener("click", function (e) {
+         const email = document.getElementById('email').value
+
+  if (email === '') {
+    e.preventDefault()
+
+    small.innerHTML = 'e-mail field cannot be emty'
+  } else if (email !== email.toLowerCase()) {
+    e.preventDefault()
+
+    small.innerHTML =
+      'email must be lowercase'
+  } else {
+  }
+ });
```

Comment:
- [ ] I entered an email with an uppercase character (ex: Test@gmail.com) and the form was submitted successfully instead of throwing an error. Kindly make sure that your form validation works with the following rules:
- If a user submits the contact form with the email field containing **uppercase characters** an error message should show up.
- If a user submits the contact form with the email field containing **only lowercase characters** the form is sent normally.

Reply:
Please I am not certain if you tested this correctly . Below is the result when I just tested with the sample mail you provided. It will be nice if you let me see the screenshot of your view

![image](https://user-images.githubusercontent.com/43151556/157997384-65b3357b-fb6c-40b6-95ab-9bcf167906be.png)